tools/WinObjC.Language/WinObjC.Language.Packaging.targets

```diff
@@ -3,11 +3,12 @@
 
   <Target Name="AddBuiltOutput" BeforeTargets="GetPackageContents" Condition="'$(SkipGetPackageContentsForOtherPlatforms)' != 'true'">
       <ItemGroup>
-          <!-- Inlcude *all* of /include/  and /include_next/ folder.  -->
-          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*"/>
+          <!-- Include *all* of /include/, /include/xplat/ and /include_next/ folder.  -->
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\**\*.*" Exclude="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
+          <_IncludeFiles Include="$(MSBuildThisFileDirectory)..\include\xplat\**\*.*"/>
```

Comment:
I'm fine with either:
1. Leaving as is.
2. Modifying 80 files to use xplat/starboard/smarttypes.h

Please don't do the ..\..\..\tools\include\starboard.h because that would be including headers outside the package.


---
In reply to: [111221091](https://github.com/Microsoft/WinObjC/pull/2493#discussion_r111221091) [](ancestors = 111221091,111036207)